src/Symfony/Component/Validator/Constraints/Valid.php

```diff
@@ -43,4 +43,11 @@ public function addImplicitGroupName(string $group)
             parent::addImplicitGroupName($group);
         }
     }
+
+    public function __construct(array $options = null, bool $traverse = null, array $groups = null, $payload = null)
+    {
+        parent::__construct($options ?? [], $groups, $payload);
+
+        $this->traverse = $traverse ?? $this->traverse ?? true;
```

Comment:
> `$this->traverse ?? true`

can `$this->traverse` really be null?

Reply:
What about `bool $traverse = true` in the constructor instead? or using `?bool` there

Reply:
Fallback to true removed to bring it in line with https://github.com/symfony/symfony/commit/fb99eb2052523cfa2a9ba36006d020796548aaa6